docs/NEW_LANGUAGE_FEATURE_INTEGRATION_PROCESS.md

```diff
@@ -5,28 +5,32 @@ merged before following this document. This document outlines the procedures
 for integrating new language features into Checkstyle,
 by updating existing checks and creating new ones.
 
-## Check Update Procedure
+Updating check modules and creating new checks to support new language features aligns Checkstyle
+with current best practices in the Java community and contributes to
+the project's ongoing evolution.
 
-Updating check modules to support new language features aligns Checkstyle
-with current best practices in the Java community
-and contributes to the project's ongoing evolution.
-
-### Java Enhancement Proposals (JEPS)
+## Java Enhancement Proposals (JEPS)
 
 Review Java Enhancement Proposals (JEPS) related to the new feature.
 JEPS provide detailed information about the goals
 and motivations behind new language features.
 This information can help us understand the feature better and figure out
-which checks are most likely to be impacted.
+which checks are most likely to be impacted or what new checks might be needed.
```

Comment:
Not for this line, but let's add a list of few general concepts that should be considered. Think (or look) back to the tracker issues that we have created. One concept is: "What problem does this new language feature solve? Can Checkstyle suggest updating existing code to use this new feature, thus improving readability/maintainability/reducing bugs/ etc"? The examples are good, but some more abstract concepts will be more beneficial in the long run.

Reply:
done